### DISCUSSION TITLE
New translator for Italian

### DISCUSSION THREAD

**Discussion by stecchio66** [sircharlo/meeting-media-manager#674] [Translations]
I would like to help to translate M³ into a language I speak,Italian (I/it).

**Comment by mtdvlpr** [comment]
Thank you so much!

To get started, please [sign into GitLocalize](https://gitlocalize.com/) using your GitHub account, and then let me know when that's done by adding a comment in this chat. 

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by stecchio66** [reply]
Done

**Reply by mtdvlpr** [reply]
Great! I've added you to the 2 repos :)

**Comment by mtdvlpr** [comment]
There is one repo for the app itself and one repo for the documentation site. Feel free to leave the documentation for what it is now, since the app itself is the most important thing. Thanks again for your willing spirit!

**Comment by mtdvlpr** [comment]
Hi @stecchio66, how's the translating coming along? Let us know if you run into any issues!

**Reply by stecchio66** [reply]
Hi @mtdvlpr, italian translation should be complete.

**Reply by mtdvlpr** [reply]
Awesome! One thing, for the documentation, the `ref` should not be translated. It's used to connect the different translations of the same post together. I fixed it now, so if you see them in English again, it's fine and you can leave them as is. Thanks again for your hard work!

**Reply by mtdvlpr** [reply]
The new translations will be released soon!

**Reply by stecchio66** [reply]
> One thing, for the documentation, the ref should not be translated. It's used to connect the different translations of the same post together. I fixed it now, so if you see them in English again, it's fine and you can leave them as is.

Sorry for this. In the future i pay more attention.

**Comment by mtdvlpr** [comment]
Dear @stecchio66, we have migrated to Crowdin for translations. Please see https://github.com/sircharlo/meeting-media-manager/discussions/722#discussioncomment-4633980 for details.

**Comment by stecchio66** [comment]
﻿﻿﻿I have requested more time to join to the crowdin project with out reply.How can to do?Il giorno 20 gen 2023, alle ore 14:50, Manoah Tervoort ***@***.***> ha scritto:﻿
Dear @stecchio66, we have migrated to Crowdin for translations. Please see #722 (comment) for details.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
Inviato da Inviato da 

**Reply by mtdvlpr** [reply]
@stecchio66, weird. I changed something now. Could you [try again](https://crowdin.com/project/meeting-media-manager) by trying to open the Italian translations? Or alternatively send us the email or username you use for Crowdin?

**Reply by mtdvlpr** [reply]
@stecchio66  I think I sent you a message on Crowdin.

**Comment by bharatrajsingh** [comment]
Hi Guys,
Why am I a part of this?
I don't have any clue.
Can anybody please tell me?

**Reply by mtdvlpr** [reply]
No clue...must be a GitHub bug I guess. Sorry for the inconvenience!

**Reply by mtdvlpr** [reply]
I reported this potential bug at https://github.com/community/community/discussions/45307#discussion-4790270

**Comment by bharatrajsingh** [comment]
Got it Manoah Tervoort,

No worries!!
